src/studio/src/designer/frontend/app-development/test/testUtils.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { render as rtlRender } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import type { RenderOptions } from '@testing-library/react';
+import type { PreloadedState } from '@reduxjs/toolkit';
+import { rest } from 'msw';
```

Comment:
I dont think `msw` is added to this project, so this might cause issues? Not sure how strict the imports in tests are checked. Might work anyway because yarn will hoist it, not sure... 

msw is great for intercepting network requests, so we might use this later for tests I think. Suggest to remove this (and we can add it later when/if we need it), or include msw in package.json.

Reply:
Good catch, removed.